src/components/templates/Audience/Audience.scss

```
@@ -2,142 +2,143 @@
 
 $light: #ffffff;
 
+$spacing: 30px;
+
+// If you change this, make sure to also change it in Audience.tsx's SEAT_SIZE
+$seat-size: 4vh;
+
 .audience-container {
   background-attachment: fixed;
   background-position: center 46%;
+
   position: relative;
-  display: flex;
-  flex: 1;
-  -webkit-flex: 1;
-  -ms-flex: 1;
-
-  .video-container {
-    z-index: z(audience-video);
-    position: fixed;
-    left: 50%;
-    top: 28%;
-    width: 45%;
-    top: 50%;
-    left: 50%;
-
-    transform: translate(-50%, -50%);
-
-    .video {
-      position: relative;
-      width: 100%;
-      padding-bottom: 50%;
-      float: left;
-      height: 0;
-
-      .frame {
-        width: 100%;
-        height: 100%;
-        position: absolute;
-        left: 0;
-        border-width: 1px black;
-        border-radius: 15px;
-        box-shadow: 2px 2px 20px $light;
-      }
+  overflow: auto;
+  height: 100%;
+  padding: $spacing;
+  min-width: fit-content;
 
-      .aspect-16-9 {
-        max-width: 88.5%;
-        margin-left: 6.5%;
-      }
-    }
+  .video-container-wrapper {
+    position: absolute;
+    z-index: z(audience-video-container);
+    width: 100%;
+    display: flex;
+    justify-content: center;
```

Playing with the styling a bit more, this looks like it will work, and should remove the direct `top` styles needed in react.

```suggestion
  .audience-overlay {
    position: absolute;
    top: 15px; /* Not sure why this is needed for alignment.. */
    left: 0;
    
    width: 100%;
    height: 100%;
    
    display: flex;
    justify-content: center;
    align-items: center;
```

Open to why this might be a worse idea than the current method though.

---

Added here, but replaced pixels with vh
https://github.com/sparkletown/sparkle/pull/1237/commits/4f768f4803498e25ab6ed9b5c963139ab4536de2

---

Is using `vh` here correct though? I believe `15px` remained perfectly aligned regardless of the size of the browser, whereas `vh` is going to change based on the height. While I couldn't see why it was required, I feel like it is going to be a `15px` `margin`/`padding` somewhere, which is why I set it in `px`.

---

Also, the order of the keys was explicitly laid out as I wanted it, with position/top/left being together, and all of the flex stuff being together, etc. Yet that didn't translate through when you made the changes.

---

`perfectly aligned regardless of the size of the browser` it didn't though. If you make the height of the screen small enough, it broke out a little. But I don't think that it has very huge difference either case